CONTRIBUTING.md

```diff
@@ -64,11 +64,11 @@ See [invertase/melos](https://github.com/invertase/melos) for more instructions
 ```
 $ git clone git@github.com:[username]/amplify-flutter.git
 $ cd amplify-flutter
-$ flutter pub global activate melos
+$ dart pub global activate melos
 $ melos bootstrap
 ```
 
-> Note: If you don't include `melos` on your path, you may execute `flutter pub global run melos bootstrap` instead of the last command above.
+> Note: If you don't include `melos` on your path, you may execute `dart pub global run melos bootstrap` instead of the last command above.
```

Comment:
```suggestion
> Note: If you're not able to run `melos`, follow [these instructions](https://dart.dev/tools/pub/cmd/pub-global#running-a-script-from-your-path) for updating your PATH to include Dart's package cache. Alternatively, you may execute `dart pub global run melos bootstrap` instead of `melos bootstrap`.
```

Reply:
`dart pub global activate melos` returns a warning message on how to update the PATH as needed. so we may skip adding it into this doc. wdyt?
this is the dart pub warning message on how to update PATH:
```
Warning: Pub installs executables into $HOME/.pub-cache/bin, which is not on your path.
You can fix that by adding this to your shell's config file (.bashrc, .bash_profile, etc.):

  export PATH="$PATH":"$HOME/.pub-cache/bin"
```

Reply:
Brilliant! Love that they've added that. Works for me to delete this warning then 👍